### PR TITLE
Use css grid to display sponsors

### PR DESCRIPTION
**Pull request by Jamdan2**
For now the names are still present, but the goals of this pr are to 

- ditch those and link the images to the websites
- center logos vertically in grid cells (they are not currently)

feel free to add anything else relevant to list

### CODE REVIEW

**Comment by andrewda** [comment]
Link for reference: https://deploy-preview-43--sert.netlify.com/sponsors/

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 source 'http://rubygems.org'
 gem 'github-pages'
 gem 'jekyll'
+gem 'tzinfo-data'
```
Comment (andrewda) [inline Gemfile]: What's this gem for?

Comment (Jamdan2) [reply Gemfile]: 'jekyll serve' wouldn't work till I added that

**Comment by Jamdan2** [comment]
🎉 👍

**Pull request merged**